### PR TITLE
[Term Entry] Pandas DataFrame `isna()`

### PR DESCRIPTION
**Pull request by astro-satyam75**
### Description

Added `.isna()` to the terms folder inside the dataframe folder. 

In the isna.md file, added the definition, description, syntax and example use cases for the isna() method in pandas.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:16]:
```suggestion
The **`.isna()`** method checks whether the objects of a `DataFrame` or a series contain missing or null values `(NA, NaN)` and returns a new object with the same shape as the original object but with boolean values `True` or `False` as the elements. `True` indicates the presence of null or missing values and `False` indicates otherwise. The original `DataFrame` object, used to call the method, remains unchanged.'
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:30]:
```suggestion
```
Remove extra line break

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:42]:
```suggestion
```
Remove extra line break

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:71]:
```suggestion
```
Remove extra line break

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:25]: Are we sure about these parameters? They generate errors for me and I can't find them in the official Pandas documentation: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.isna.html#pandas.DataFrame.isna

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:73]: This doesn't work in Python v3.9.5, are we sure that `.isna()` has these parameters?

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:102]:
```suggestion
### Example 3: Series without optional parameters
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:116]:
```suggestion
The Output from using `.isna()` on Series without optional parameters:
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:127]: Again, with parameters does not work in Python 3.9.5.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@astro-satyam75 I did a first review and left some comments. The area of most concern is that the info on parameters seems to be inaccurate, and the examples with parameters fail to work in Python v3.9.5

**Comment by astro-satyam75** [comment]
Upon reviewing the latest official documentation and updating my pandas library to the latest version, I was getting the same errors. The optional parameters were removed after the v1.0.0 of the pandas library.

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:14]:
```suggestion
```
remove extra line break

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:16]:
```suggestion
The **`.isna()`** method checks whether the objects of a dataframe or a series contain missing or null values `(NA, NaN)` and returns a new object with the same shape as the original object but with boolean values `True` or `False` as the elements. True indicates the presence of null or missing values and False indicates otherwise. The original `DataFrame` object, used to call the method, remains unchanged.
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:26]:
```suggestion
Here are some examples for using the `.isna()` method on both DataFrame and Series objects:

```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:27]:
```suggestion
### Example 1: `.isna()` with DataFrame

```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:53]:
```suggestion

### Example 2: `.isna()` with Series

```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:65]:
```suggestion

The Output from using `.isna()` on Series:

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@astro-satyam75 I left a few more comments. Note, you need a line break before and after headers and before and after code blocks, and you shouldn't have multiple line breaks in a row, only one line break should separate items.

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:14]:
```suggestion

The **`.isna()`** method checks whether the objects of a `DataFrame` or a series contain missing or null values `(NA, NaN)` and returns a new object with the same shape as the original object but with boolean values `True` or `False` as the elements. `True` indicates the presence of null or missing values and `False` indicates otherwise. The original `DataFrame` object, used to call the method, remains unchanged.
```
Ad line break after metadata.

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:22]:
```suggestion

## Example
```
Line break before header.

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:25]:
```suggestion

### Example 1: `.isna()` with DataFrame

```
Line break before and after header.

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:51]:
```suggestion

### Example 2: `.isna()` with Series

```
Line break before and after header.

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:63]:
```suggestion

The Output from using `.isna()` on Series:

```
Line breaks before and after code blocks.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@astro-satyam75 A few more comments, mostly around line breaks.

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:14]:
```suggestion
```
Still need to remove extra line break.

**Review by SSwiniarski — APPROVED** [review]
With the exception of one additional line break that needs to be removed, looks good for a second review. Just an FYI- we monitor a lot of PRs, so tagging us in a comment when you're done with review comments helps us get back to you quicker.

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:3]:
```suggestion
Description: 'Checks whether the objects of a DataFrame or a Series contain missing or null values and returns a corresponding boolean value.'
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:16]:
```suggestion
The **`.isna()`** method checks whether the objects of a DataFrame or a Series contain missing or null values `(NA, NaN)` and returns a new object with the same shape as the original but with boolean values `True` or `False` as the elements. `True` indicates the presence of null or missing values and `False` indicates otherwise. The original DataFrame object, used to call the method, remains unchanged.
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:47]:
```suggestion
The code above results in the following output:
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:71]:
```suggestion
The code above results in the following output:
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:29]:
```suggestion
### Example 1: `.isna()` with a DataFrame
```

File: content/pandas/concepts/dataframe/terms/isna/isna.md
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/isna/isna.md:57]:
```suggestion
### Example 2: `.isna()` with a Series
```

**Comment by astro-satyam75** [comment]
@caupolicandiaz Thanks for the suggested changes. I've added the correction you've requested

**Review by caupolicandiaz — APPROVED** [review]
@astro-satyam75 Looks good. I just made a few additional edits to update the metadata and fix a spacing issue.

**Comment by yangc95** [comment]
🙌 @astro-satyam75 here's the live link: https://www.codecademy.com/resources/docs/pandas/dataframe/isna

Screenshot 2023-04-21 at 7 45 19 PM

cc: @caupolicandiaz @SSwiniarski @KTom101

**Pull request merged**
